### DISCUSSION TITLE
ImageAndMeshViewer

### DISCUSSION THREAD

**Discussion by AkagiVoid** [nesrak1/UABEA#277] [Ideas]
I would like a view port as well that can render Texture2Ds and Meshs. ^w^
Is using "SB3Utility V23.1.7" (this version can only be found by DL Koikatsu ~~Better Repack~~) to view the images and find the ones I need/want to change. >w<

**Comment by nesrak1** [comment]
That's what this is supposed to be: https://github.com/nesrak1/UABEANext

For now though, you'd probably want something like AssetRipper or AssetStudio.

**Comment by AkagiVoid** [comment]
> That's what this is supposed to be: https://github.com/nesrak1/UABEANext
> 
> For now though, you'd probably want something like AssetRipper or AssetStudio.

Yea, just tried dat out, and its buggy, also features are missing as well like "Filter" to exclude out things I don't need, and when I went to press "Edit Data" when i didn't select anything down below 2 edit, the tool would crash. X3
And "Plugins" don't seem to do anything yet. X3

And for personal reference I would use AssetStudio it seems to have a much friendlier UI layout compared to AssetRipper that bombards you with the settings menu r8t out of the gate. X3

**Reply by nesrak1** [reply]
UABEANext is the development version of UABEA, so of course it has bugs, it's not even finished yet. I was just letting you know that the ability to preview Meshes, Texture2Ds, etc. is part of the new version. As the README implies, I don't expect people to use it as a replacement to UABEA. Hence my reply "For now though, you'd probably want..."

Sadly, development doesn't happen overnight. If you want to edit support of UABEA but the preview support of AssetStudio in a bug-free and complete program, you won't find it.
